### DISCUSSION TITLE
server Url

### DISCUSSION THREAD

**Discussion by shoaibmalik2611** [saran-surya/email_auth_node#23] [Q&A]
Hey, Where from i get server url and key?

**Comment by saran-surya** [comment]
> Hey, Where from i get server url and key?

Please read the step by step guide, it should help
[https://saran-surya.github.io/email-auth-node/](https://saran-surya.github.io/email-auth-node/)

**Reply by saran-surya** [reply]
Hi @shoaibmalik2611 is your issue resolved ?
